### PR TITLE
Rephrase some UI labels and tooltips.

### PR DESCRIPTION
**Pull request by vincent-unity**
Rephase some UI labels and tooltips.

### CODE REVIEW

File: com.unity.mobile.notifications/Editor/NotificationSettingsManager.cs [hunk 1]
```diff
@@ -175,117 +171,98 @@ internal static NotificationSettingsManager Initialize()
...
 
             var iosSettings = new List<NotificationSetting>()
             {
                 new NotificationSetting(
                     "UnityNotificationRequestAuthorizationOnAppLaunch",
                     "Request Authorization on App Launch",
-                    "It's recommended to make the authorization request during the app's launch cycle. If this is enabled the user will be shown the authorization pop-up immediately when the app launches. If it’s unchecked you’ll need to manually create an AuthorizationRequest before your app can send or receive notifications.",
-                    notificationSettingsManager.GetiOSNotificationSettingsValue<bool>(
-                        "UnityNotificationRequestAuthorizationOnAppLaunch", true),
+                    "It's recommended to make the authorization request during the app's launch cycle. If this is enabled the authorization pop-up will show immediately during launching. Otherwise you need to manually create an AuthorizationRequest before sending or receiving notifications.",
```
Comment (aurimasc) [inline com.unity.mobile.notifications/Editor/NotificationSettingsManager.cs]: "authorization pop-up will show up immediately" I think (missing UP)

File: com.unity.mobile.notifications/Editor/NotificationSettingsManager.cs [hunk 2]
```diff
@@ -175,117 +171,98 @@ internal static NotificationSettingsManager Initialize()
...
                         ),
                         new NotificationSetting(
                             "UnityRemoteNotificationForegroundPresentationOptions",
                             "Remote Notification Foreground Presentation Options",
-                            "The default presentation options for received remote notifications. In order for the specified presentation options to be used your app must had received the authorization to use them (the user might change it at any time). ",
-                            notificationSettingsManager
-                                .GetiOSNotificationSettingsValue<PresentationOption>(
-                                "UnityRemoteNotificationForegroundPresentationOptions",
-                                (PresentationOption)iOSPresentationOption.All)
+                            "Configure the default presentation options for received remote notifications. In order to use the specified presentation options, your app must have received the authorization(the user might change it at any time).",
```
Comment (aurimasc) [inline com.unity.mobile.notifications/Editor/NotificationSettingsManager.cs]: Missing spece before opening paren.

Comment (vincent-unity) [reply com.unity.mobile.notifications/Editor/NotificationSettingsManager.cs]: Fixed at https://github.com/Unity-Technologies/com.unity.mobile.notifications/pull/24/commits/c6a9942c5450408065da1912e6191c4f9c689378.

File: com.unity.mobile.notifications/Editor/NotificationSettingsManager.cs (hunk 1 above)
Comment (vincent-unity) [reply com.unity.mobile.notifications/Editor/NotificationSettingsManager.cs]: Fixed at https://github.com/Unity-Technologies/com.unity.mobile.notifications/pull/24/commits/c6a9942c5450408065da1912e6191c4f9c689378.

**Pull request merged**